metadata.rb

```diff
@@ -1059,6 +913,7 @@
 '
 )
 depends 'nfs', '~> 2.1.0'
+depends 'line', '~> 0.6.3'
```

Comment:
dependency issue where we need to pin the version of a dependent cookbook for compatibility reasons